espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/handlers/SetClipboard.java

```diff
@@ -71,7 +71,7 @@ public void run() {
                 default:
                     throw new IllegalArgumentException(
                             String.format("Only '%s' content types are supported. '%s' is given instead",
-                                    ClipboardDataType.supportedDataTypes(), contentType));
+                                    ClipboardDataType.Companion.supportedDataTypes(), contentType));
```

Comment:
how about JvmStatic usage?

Reply:
Every java file will eventually be ported over to Kotlin so I don't think JvmStatic is necessary. I'm pretty sure it's only necessary in the event that we're calling Kotlin from Java.

https://stackoverflow.com/questions/48780003/why-and-when-to-use-jvmstatic-with-companion-objects

http://kotlinlang.org/docs/reference/java-to-kotlin-interop.html#static-methods